src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -932,6 +933,9 @@ static int parse_args(vector<const char*>& args, std::ostream *err_msg, Config *
   for (i = args.begin(); i != args.end(); ) {
     if (ceph_argparse_flag(args, i, "-h", "--help", (char*)NULL)) {
       return -ENODATA;
+    } else if (ceph_argparse_flag(args, i, "-v", "--version", (char*)NULL)) {
+      std::cout << pretty_version_to_str() << std::endl;
+       _exit(0);
```

Comment:
I don't understand why an error is returned for --help. As per the convention followed here, the option --version also should return an error. But Why can't they return a positive int values to distinguish --help and --version ? Then process inside rbd_nbd() and then return 0 from there ?